src/UITableView/UITableView+Utils.m

```diff
@@ -0,0 +1,48 @@
+
+#import "UITableView+Utils.h"
+#import "UIView+IPAncestry.h"
+
+@implementation UITableView (Utils)
+
+- (void)ip_setZeroSeparatorInsets {
+    if ([self respondsToSelector:@selector(setSeparatorInset:)]) {
+        [self setSeparatorInset:UIEdgeInsetsZero];
+    }
+
+    if ([self respondsToSelector:@selector(setLayoutMargins:)]) {
+        [self setLayoutMargins:UIEdgeInsetsZero];
+    }
```

Comment:
separator insets are controlled by layoutmargins in later versions of OS. Suggest a rename?